src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -58,17 +60,24 @@ object WdlExpression {
           case _ => throw new WdlExpressionException(s"Invalid operator: ${a.getName}")
         }
       case a: Ast if a.getName == "MemberAccess" =>
-        val lhs = evaluate(a.getAttribute("lhs"), lookup, functions).map {
-          case x: WdlObject => x
-          case _ => throw new WdlExpressionException("Left-hand side of expression must be a WdlObject")
-        }
         val rhs = a.getAttribute("rhs") match {
           case x:Terminal if x.getTerminalStr == "identifier" => x.getSourceString
           case _ => throw new WdlExpressionException("Right-hand side of expression must be identifier")
         }
-        lhs.map { _.value.getOrElse(rhs, throw new WdlExpressionException(s"Could not find key $rhs")) }
+        val lhs = evaluate(a.getAttribute("lhs"), lookup, functions).map {
+          case o: WdlObject => o.value.getOrElse(rhs, throw new WdlExpressionException(s"Could not find key $rhs"))
+          case ns: WdlBinding =>
+            lookup(ns.namespace match {
```

Comment:
`ns.namespace map {n => s"$n.$rhs"} getOrElse rhs` (might not be 100% syntactically correct)